pkg/pods/pod.go

```diff
@@ -292,17 +239,7 @@ func (pod *Pod) Uninstall() error {
 		}
 	}
 
-	// remove runit services
-	sbTemplate := runit.NewSBTemplate(pod.Id)
-	err = pod.ServiceBuilder.Remove(sbTemplate)
-	if err != nil {
-		return err
-	}
-
-	_, err = pod.ServiceBuilder.Rebuild()
-	if err != nil {
-		return err
-	}
+	// remove runit services here
```

Comment:
whoops, actually not necessary. When the launchable is halted, it also prunes its runit service. I accidentally forgot to remove the comment, I'll nuke that.